crypto/cmd/cmd.go

```diff
@@ -106,17 +104,18 @@ func Cmd() *cobra.Command {
 			cmd.Println(publicKeyAsPEM)
 		},
 	})
-
+	cmd.PersistentFlags().AddFlagSet(core.NewClientConfig().FlagSet())
```

Comment:
Only the crypto command adds the flagset at this point. The vdr and network do not add it here. Why the difference?

Reply:
good spot, leftover from reverted refactor